ember-app/app/components/hb-markdown-composer.js

```diff
@@ -37,8 +37,15 @@ var HbMarkdownComposerComponent = Ember.Component.extend({
       fd.append('policy', response.policy);
       fd.append('signature', response.signature);
       fd.append('success_action_status', "201");
-      fd.append('file', file);
 
+      if(HUBOARD_ENV.FEATURES.ENCRYPTED_UPLOADS) {
```

Comment:
As more generally commented, it seems to me that this could be:

```javascript
if(HUBOARD_ENV.CONFIG && HUBOARD_ENV.CONFIG.AWS_S3_ENCRYPTED === 'true')
...
```